lms/tasks/email_digests.py

```diff
@@ -158,7 +170,10 @@ def send_instructor_email_digest(
             digest_service = request.find_service(DigestService)
 
             digest_service.send_instructor_email_digest(
-                h_userid, created_after, created_before, **kwargs
+                h_userid=h_userid,
+                created_after=created_after,
+                created_before=created_before,
+                **kwargs,
```

Comment:
Changing these to keyword arguments just to avoid any potential confusion about argument order (especially `created_after` and `created_before`)